scripts/dts/python-devicetree/src/devicetree/edtlib.py

```diff
@@ -3210,16 +3227,23 @@ def _phandle_val_list(
 
     return res
 
+def _address_cells_self(node: Optional[dtlib_Node]) -> Optional[int]:
+    # Returns the #address-cells setting for 'node', giving the number of <u32>
+    # cells used to encode the address in the 'reg' property
```

Comment:
This comment is the same as for `_address_cells()`. Maybe clarify how this new function differs? (own property vs. parent bus property)

Reply:
Actually the comment should be fixed for `_address_cells()`, since it returns the parent's `#address-cells` :). But the comment for `own_address_cells()` is correct: it returns the value for the node, or `None` if it doesn't have one.